src/Database/Auth/ApiKeyAuthenticator.ts

```diff
@@ -69,7 +70,7 @@ export class ApiKeyAuthenticator {
 
     return tryAuthenticate()
       .then((response: IResponse) => {
-        const body = response.body;
+        const body = response.body as IHash;
```

Comment:
just for internal use. object that can contain any properties.
if use Object type, we will have compiling errors trying to access properties (because any object property should be defined in ts).